packages/dataviews/src/utils.ts

```diff
@@ -22,14 +23,24 @@ export function sanitizeOperators< Item >( field: NormalizedField< Item > ) {
 		ALL_OPERATORS.includes( operator )
 	);
 
+	// Don't allow between operator if the field has elements.
+	if ( field.elements && operators.includes( OPERATOR_BETWEEN ) ) {
+		operators = operators.filter(
+			( operator ) => operator !== OPERATOR_BETWEEN
+		);
+	}
+
 	// Do not allow mixing single & multiselection operators.
 	// Remove multiselection operators if any of the single selection ones is present.
+	// The 'Between' operator is unique as it can be combined with single selection operators.
 	const hasSingleSelectionOperator = operators.some( ( operator ) =>
 		SINGLE_SELECTION_OPERATORS.includes( operator )
 	);
-	if ( hasSingleSelectionOperator ) {
+	if ( hasSingleSelectionOperator || operators.includes( OPERATOR_BETWEEN ) ) {
```

Comment:
Why do we want to remove any multi-selection operator if the between operator is present?

Reply:
Hmm, you're right. The between operator can be applied when a multi-selection operator is present. Updated the logic in 355177da4ad2c6d8f2eb17bfd98ee5455dc64da5.